block_iagora.php

```diff
@@ -61,4 +69,27 @@ public function get_content() {
     public function applicable_formats() {
         return ['all' => true];
     }
+     
+    /**  
+     * Allow the block to be configured.
+     * 
+    * @return bool
+    */
```

Comment:
```suggestion
    /**  
     * Allow the block to be configured.
     * 
     * @return bool
     */
```
Missing spaces for indentation?